### PR TITLE
dev-to-prod

### PR DESCRIPTION
**Pull request by Izobov**
Please `click on` the `(^) Preview` tab and select the appropriate sub-template:

- [Pull-Request Template into branch:dev](?expand=1&template=pr_to_dev.md)
- [Pull-Request Template into branch:main](?expand=1&template=pr_to_main.md)

### CODE REVIEW

File: src/lib/components/_main_/header_redisigned/UserDropdownPopup.svelte [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  // │ Please, structure the imports as follows:                              │
+  // │ 1. svelte/sveltekit imports                                            │
+  // │ 2. project-internal files and logic                                    │
+  // │ 3. component import(s)                                                 │
+  // │ 4. assets import(s)                                                    │
+  // │ 5. type(s) imports(s)                                                  │
+  // ╰────────────────────────────────────────────────────────────────────────╯
+
+
+
```
Comment (migbash) [inline src/lib/components/_main_/header_redisigned/UserDropdownPopup.svelte:18]: Unecessary number of newline spaces.

File: src/lib/components/_main_/header_redisigned/UserDropdownPopup.svelte [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  import { page } from "$app/stores";
+  import userBetarenaSettings from "$lib/store/user-settings.js";
+  import TranslationText from "$lib/components/misc/Translation-Text.svelte";
+  import { logoutUser } from "$lib/utils/user";
+  import { fly } from "svelte/transition";
+  import { scoresNavbarStore } from "./_store";
+  import type { B_NAV_T } from "@betarena/scores-lib/types/navbar.js";
+ // #endregion ➤ 📦 Package Imports
+  $: translationData = $page.data.B_NAV_T as B_NAV_T | null | undefined;
```
Comment (migbash) [inline src/lib/components/_main_/header_redisigned/UserDropdownPopup.svelte:28]: Should be inside the `// #region ➤ 📌 VARIABLES`

**Review by migbash — CHANGES_REQUESTED** [review]
Some stuff to reveiw @Izobov

File: src/lib/components/_main_/header_redisigned/SportNavigation/Header-C-Bookmakers.svelte [hunk 3]
```diff
@@ -0,0 +1,493 @@
+<!--
+╭──────────────────────────────────────────────────────────────────────────────────╮
+│ 📌 High Order Component Overview                                                 │
+┣──────────────────────────────────────────────────────────────────────────────────┫
+│ ➤ Internal Svelte Code Format :|: V.8.0                                          │
+│ ➤ Status :|: 🔒 LOCKED                                                           │
+│ ➤ Author(s) :|: @migbash                                                         │
```
[the remark this answers is not in the document]
Comment (Izobov) [reply src/lib/components/_main_/header_redisigned/SportNavigation/Header-C-Bookmakers.svelte:7]: actualy it's fully your code, I just moved it to another folder, as we can drop folder with prev header

File: src/lib/components/page/lobby/highlights/Highlights-Widget.svelte [hunk 4]
```diff
@@ -276,6 +276,7 @@
   -->
   <svelte:component
     this={HighlightsGridDynamic}
+    on:showModal
```
[the remark this answers is not in the document]
Comment (Izobov) [reply src/lib/components/page/lobby/highlights/Highlights-Widget.svelte:279]: it's in use.  Inside GridComponent ithere is a dispatch

File: src/lib/components/_main_/header_redisigned/Header-Nav-Btn.svelte
[the remark this answers is not in the document]
Comment (Izobov) [reply src/lib/components/_main_/header_redisigned/Header-Nav-Btn.svelte:1]: I didn't have time to configurate Prettier. Will do after this task

[the remark this answers is not in the document]
Comment (Izobov) [reply src/lib/components/_main_/header_redisigned/Header-Nav-Btn.svelte:1]: and it's looks like eslint dosent work for me

[the remark this answers is not in the document]
Comment (migbash) [reply src/lib/components/_main_/header_redisigned/Header-Nav-Btn.svelte:1]: Closing review, will resolve for next `PR`.

File: src/lib/components/page/lobby/highlights/Highlights-Widget.svelte (hunk 4 above)
[the remark this answers is not in the document]
Comment (migbash) [reply src/lib/components/page/lobby/highlights/Highlights-Widget.svelte:279]: Sure, but it's not doing anything this `on:showModal`.

If it needs to do something, it should be `on:showModal={/* logic here */`}` ?

**Review by jonsnowpt — APPROVED** [review]

**Pull request merged**